draft-ietf-privacypass-protocol.md

```diff
@@ -221,7 +226,7 @@ with the input challenge and Issuer key identifier as described below:
 ~~~
 nonce = random(32)
 challenge_digest = SHA256(challenge)
-token_input = concat(0x0001,
+token_input = concat(0x0001, // 2-byte length
```

Comment:
I find this a bit confusing — it almost sounds like there would be an extra 2-byte length field before this?

How about:

```suggestion
token_input = concat(0x0001, // Token type field is 2 bytes long
```